packages/hop-node/src/watchers/BondTransferRootWatcher.ts

```diff
@@ -41,7 +43,24 @@ class BondTransferRootWatcher extends BaseWatcher {
   }
 
   async checkTransfersCommittedFromDb () {
-    const dbTransferRoots = await this.db.transferRoots.getUnbondedTransferRoots()
+    let filterSourceChainId: number | undefined
```

Comment:
Does it make sense to DRY these functions up and put them in BaseWatcher.ts? I think the destination ones make sense since they are identical, AFAICT. Something like:

`const filterSourceChainId = getFilterSourceChainId()`

&

`const filterDestinationChainIds = getFilterDestinationChainIds()`

Reply:
Will do

Reply:
Resolved with https://github.com/hop-protocol/hop/commit/581ed82661c82a7586482afc5f7a631dab7ed18f